style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  margin: unset;
+  overflow: auto;
+}
+
+.menu {
+  display: flex;
+  justify-content: flex-end;
+  padding: 5%;
+  margin-bottom: 30px;
+}
+
+.job {
+  color: #36b37e;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+}
+
+.button {
+  color: #172b4d;
+  margin-top: 100px;
+  justify-content: center;
+  display: flex;
+}
+
+.greeting {
+  justify-content: center;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #172b4d;
+}
+
+.media {
+  display: flex;
+  gap: 1rem;
+  justify-content: center;
+  list-style-type: none;
+  margin-top: 24px;
+}
+
+.headline {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 20px;
+  text-align: center;
+  background: url('images/header.svg') no-repeat center;
```

Comment:
- [ ] Lastly, It looks like your background image is missing one of its contents. From the images below, you can see that the part I've marked out on the Figma design is present in the Figma design but isn't showing on yours. Kindly fix that as well so your design looks like an exact replica of the Figma design :100:  

| Your Design | Figma Design |
|--------|--------|
| ![image](https://github.com/KiengPhyl/Philemon-portfolio/assets/102699958/bc914af7-ab61-43c5-bc96-01c26374b7e1) | ![image](https://github.com/KiengPhyl/Philemon-portfolio/assets/102699958/5a3359a6-a710-4e9a-98ae-7f089a82413d) |